CHANGELOG.md

```diff
@@ -14,10 +14,11 @@ The format is based on [Common Changelog](https://common-changelog.org).
 
 ### Changed
 
-- **Breaking**: Change `permit2Params` to `transferData`, which has type `bytes`
-  ([#161](https://github.com/sablier-labs/v2-periphery/pull/161))
+- Install `permit2` depencency from Uniswap repository ([#198](https://github.com/sablier-labs/v2-periphery/pull/198))
```

Comment:
Not ordered alphabetically

Reply:
Why does it have to alphabetically? For it makes more sense to order chronologically

Reply:
The common changelog standard doesn't require alphabetical order

Reply:
Not all entries will have associated PRs so it will be difficult to remember the order in which the changes had been implemented

Reply:
chronologically, i.e. the order the PRs were created, in this case: 
- 198
- 191
- 187
- 161

Reply:
But I agree **Breaking** should be at top

Reply:
@andreivladbrg we have PRs for each change, but this may not always be the case. See for example the [PRBMath changelog](https://github.com/PaulRBerg/prb-math/blob/main/CHANGELOG.md).

Can we please use alphabetic ordering?